### ISSUE TITLE
Uplink: Hacker Elite

### ISSUE THREAD

**Issue by talraash** [ptitSeb/box86-compatibility-list#2]
### Description

```
Game works fine, with good speed.
```

### Tested on

Raspberry pi 4
### Status
**Previous behavior:** 

**Current behavior:**

Works fine

### Screenshots

![Screenshot_2020-04-27_11-45-04](https://user-images.githubusercontent.com/22260439/80352555-b90a3e80-887c-11ea-9e35-d0ecee960206.png)

**Comment by talraash** [comment]
Is that correct use OpenGL tag if i don't use gl4es wrapper to run?

**Comment by ptitSeb** [comment]
Yes, without the need of gl4es, and without speicfic hack, a game is probably using OpenGL 1.x or OpenGL 2.x (or no opengl at all)

From memory, I think this one is OpenGL 1.x, but I'm unsure

**Comment by talraash** [comment]
I think that the label "Working perfectly" should be used for games with several reports with different hardware, and by default it should be set to "Works". For example, some games on rpi4 may have 60fps, and on sbc with allwinner h3 <30.

**Comment by ptitSeb** [comment]
Ok (but for this particular game, 30fps would still qualify it to "Working perfectly").

I'll create a Taks "Works" and I'll give the "Working perfectly" label to games that have been tested successfully on more than 1 platform.
